converters/fromZigbee.js

```diff
@@ -2736,7 +2736,7 @@ const converters = {
             }
             const piHeatingDemand = msg.data['pIHeatingDemand'];
             if (typeof piHeatingDemand == 'number') {
-                result.operation = piHeatingDemand >= 10 ? 'heating' : 'idle';
```

Comment:
This will be a breaking change, so please keep `operation` in for now and add `DEPRECATED` above it.

Reply:
Done